includes/configs/taxonomies.php

```diff
@@ -15,6 +15,7 @@
 		'post_type'         => [ 'listing' ],
 		'hierarchical'      => true,
 		'show_admin_column' => true,
+		'meta_box_cb'       => false,
```

Comment:
Please make sure that this also works for vendor, request, booking and user categories (if they have ones), maybe this parameter is needed for all the category configs.

Reply:
In which way need to add this parameter for all post-based models: manually for all configs or maybe through loop somewhere in the code?

Reply:
You can add it manually for now (also in Bookings and Requests extensions), we'll unify all the base configs and move them to Attribute in future updates